src/geojson/FeatureCollection.cpp

```diff
@@ -7,15 +7,75 @@ Feature FeatureCollection::get_feature(int index) const {
     return features[index];
 }
 
+int FeatureCollection::find(Feature feature) {
```

Comment:
This could be written as
```
FeatureList::iterator it = std::find(features.begin(), feature.end(), feature)
return std::distance(feature.begin(), it);
```